packages/internal/src/utils.js

```diff
@@ -269,6 +269,10 @@ export const synchronizedTee = (sourceStream, readerCount) => {
       [Symbol.asyncIterator]() {
         return reader;
       },
+      // eslint-disable-next-line no-restricted-globals
+      [Symbol.asyncDispose]() {
+        throw Error('asyncDispose not implemented.');
+      },
```

Comment:
According to the [proposal](https://arai-a.github.io/ecma262-compare/?pr=3000&id=sec-%25asynciteratorprototype%25-%40%40asyncdispose), on async iterators this is the equivalent of calling `return()`.

```suggestion
      async [Symbol.asyncDispose]() {
        await reader.return();
      },
```